addon/mixins/resize-text.js

```diff
@@ -1,13 +1,31 @@
 import Mixin from '@ember/object/mixin';
 import { inject as service } from '@ember/service';
 
+const extractPixels = function(value) {
+  let match = /(\d+)px/i.exec(value || '');
+  return match ? parseInt(match[1], 10) : 0;
+};
+
 export default Mixin.create({
 
   classNames: ['resize-text'],
 
   minSize: 2,
   maxSize: 80,
 
+  // NOTE: shrink and grow are shortcuts (as mentioned below)
+  // the reasoning: often times the template developer does not have access to (or does not know)
+  // the specified font size (in pixels)
+  // obviously that can be computed by you, the developer, but these shortcuts handle it for you
```

Comment:
I am not sure if I understand this correctly. If the dev does not know the initial font size, how would he be able to calculate constraints based on that font size? Or is this an entirely different use case?

Reply:
I think the idea I had in mind was that you may have a separation between the developer writing the structure of the application (using handlebars and the resize-text component); a separate developer may be defining the styles (which may also change based on media queries, i.e. responsive). Of course this is a custom use-case, and may not be applicable to most applications, but the idea was that I may only know (as the HTML/template dev) what the maximum width of my current container is... and let the font resize accordingly/automatically..